lms/static/certificates/sass/_components.scss

```diff
@@ -569,3 +572,89 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.5); /*dim the background*/
```

Comment:
Please use colors from our Pattern Library palette - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_variables.scss#L70. You can see an example of how their referenced using a Sass function here - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/lms%2Fstatic%2Fcertificates%2Fsass%2F_components.scss#L81